ios/PolyPodApp/Features/Feature.swift

```diff
@@ -28,7 +29,9 @@ class Feature {
         name = translations?.name ?? manifest.name ?? id
         author = translations?.author ?? manifest.author
         description = translations?.description ?? manifest.description
-        primaryColor = parseColor(hexValue: translations?.primaryColor ?? manifest.primaryColor)
+        let primaryColor = parseColor(hexValue: translations?.primaryColor ?? manifest.primaryColor)
+        self.primaryColor = primaryColor
+        thumbnailColor = parseColor(hexValue: translations?.thumbnailColor ?? manifest.thumbnailColor) ?? primaryColor
```

Comment:
And there's no self.thumbnailColor assignment here.

Reply:
It's implicit, I only have to spell out `self.primaryColor` because I also created a local variable called `primaryColor` to get around that Xcode 12 compile error.

Reply:
As for `primaryColor` being substituded by `thumbnailColor`: That's only for the feature list card background. `primaryColor` is still being used for the nav bar colour.